<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditElipse</class>
 <widget class="QMainWindow" name="EditElipse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Elipse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEditOkEllipse">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinEditYEllipse">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Set X for elipse's center (in pixels)</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>720</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinEditXEllipse">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Set X for elipse's center (in pixels)</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1280</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinEditAEllipse">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Set radius A for elipse (in pixels)</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinEditBEllipse">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Set radius B for elipse (in pixels)</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Ellipse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>210</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Color</string>
    </property>
    <widget class="QSpinBox" name="spinEditRedEllipse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Set X for circle's center (in pixels)</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinEditGreenEllipse">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Set X for circle's center (in pixels)</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinEditBlueEllipse">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Set X for circle's center (in pixels)</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboEllipseName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Thickness : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinEditEllipseThickness">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <zorder>btnEditOkEllipse</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>spinEditXEllipse</zorder>
   <zorder>spinEditYEllipse</zorder>
   <zorder>label_3</zorder>
   <zorder>spinEditAEllipse</zorder>
   <zorder>label_4</zorder>
   <zorder>spinEditBEllipse</zorder>
   <zorder>label_5</zorder>
   <zorder>groupBox</zorder>
   <zorder>comboEllipseName</zorder>
   <zorder>label_9</zorder>
   <zorder>spinEditEllipseThickness</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
